src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.exception.ExceptionMessages;
+import duke.task.TaskList;
+
+public class Duke {
+
+    public static void run() {
```

Comment:
Providing comments to describe what your method is doing would be more helpful to the reader.